site/en/guide/_mnist_tf2.ipynb

```diff
@@ -0,0 +1,316 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### Copyright 2018 The TensorFlow Authors.\n",
+    "\n",
+    "Licensed under the Apache License, Version 2.0 (the \"License\").\n",
+    "\n",
+    "# MNIST, with TensorFlow 2.0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook trains a simple MNIST model, demonstrating a basic workflow using TensorFlow 2.0 APIs.\n",
+    "\n",
+    "The basic workflow consists of:\n",
+    "\n",
+    "- defining a model\n",
+    "- preprocessing your data into a tf.data.Dataset\n",
+    "- training over a dataset\n",
+    "  - using tf.GradientTape to compute gradients\n",
+    "  - using stateful tf.metrics.* APIs to record metrics of interest\n",
+    "  - logging those metrics with tf.summary.* APIs so that they can be viewed in TensorBoard\n",
+    "  - using tf.train.Checkpoint to save and restore weights\n",
+    "- export a SavedModel using tf.saved_model (This SavedModel is a portable representation of the model, and can be imported into C++, JS, Python without knowledge of the original TensorFlow code.)\n",
+    "- reimport that SavedModel and demonstrate its usage in Python."
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Please add a nightly install so this can be run in colab.

!pip install tf-nightly-2.0-preview